tests/nnapi/specs/V1_2/reduce_mean_dynamic_1_nnfw.mod.py

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright (C) 2018 The Android Open Source Project
+# Copyright (C) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+'''
+Testing ReduceSum op when the input is dynamic.
+    input[1, 3, 4, 1] shape [4]  (value of shape will be [1, 3, 4, 1])
+          |             |
```

Comment:
is it correct expression that "value of shape is [1,3,4,1] ?

Reply:
modified it.